src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+ const cleanedUpString = sourceString
+    .split('\n')
+    .join('')
+    .trim()
+    .split(';')
+    .map(el => el.split(':'));
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Ensure that each method in the chain starts on a new line and is indented to the same level.